scripts/gtk-bundle-from-msys2.sh

```diff
@@ -58,25 +58,33 @@ getpkg() {
 	fi
 }
 
-GLIB=$(getpkg glib2)
-ATK=$(getpkg atk)
-PANGO=$(getpkg pango)
-CAIRO=$(getpkg cairo)
-ADW=$(getpkg adwaita-icon-theme)
-GTK=$(getpkg $gtk)
-
-cat <<EOF
-Using:
-glib: $GLIB
-atk: $ATK
-pango: $PANGO
-cairo: $CAIRO
-adwaita $ADW
-$gtk: $GTK
+read -d' ' pkgs <<EOF
+libwinpthread
+gcc-libs
+gettext
+libiconv
+bzip2
+libffi
+fontconfig
+freetype
+glib2
+atk
+pango
+cairo
+adwaita-icon-theme
+$gtk
 EOF
 
+for i in $pkgs; do
+	pkg=$(getpkg $i)
+	files="$files $pkg"
```

Comment:
no it's not, just makes things clearer and avoids any issue if this variable gets unexpectedly used before in the script, but that's unlikely so do as you like